python/ptest/cases/gomspace_checkout_case.py

```diff
@@ -0,0 +1,104 @@
+# any other Gomspace state fields.
+from .base import SingleSatOnlyCase
+
+
+class GomspaceCheckoutCase(SingleSatOnlyCase):
+
+    def setup_case_singlesat(self, simulation):
+        simulation.flight_controller.write_state(
+            "pan.state", 9)  # Manual state
+        self.run_case_singlesat(simulation)
+        print("Gomspace cases finished.")
+
+    def run_case_singlesat(self, simulation):
+        simulation.cycle_no = simulation.flight_controller.read_state(
+            "pan.cycle_no")
+
+        # readable fields
+
+        vboost = [
+            simulation.flight_controller.read_state("gomspace.vboost.output1"),
+            simulation.flight_controller.read_state("gomspace.vboost.output2"),
+            simulation.flight_controller.read_state("gomspace.vboost.output3")]
+
+        vbatt = simulation.flight_controller.read_state("gomspace.vbatt")
+        if vbatt < 6.0 or vbatt > 8.4:
+            print("Vbatt is out of expected range [6, 8.4] at: " + vbatt)
+
+        currin = [
+            simulation.flight_controller.read_state("gomspace.curin.output1"),
+            simulation.flight_controller.read_state("gomspace.curin.output2"),
+            simulation.flight_controller.read_state("gomspace.curin.output3")]
+
+        currsun = simulation.flight_controller.read_state("gomspace.cursun")
+        cursys = simulation.flight_controller.read_state("gomspace.cursys")
+
+        curout = [
+            simulation.flight_controller.read_state("gomspace.curout.output1"),
+            simulation.flight_controller.read_state("gomspace.curout.output2"),
+            simulation.flight_controller.read_state("gomspace.curout.output3"),
+            simulation.flight_controller.read_state("gomspace.curout.output4"),
+            simulation.flight_controller.read_state("gomspace.curout.output5"),
+            simulation.flight_controller.read_state("gomspace.curout.output6")]
+
+        output = [
+            simulation.flight_controller.read_state("gomspace.output.output1"),
+            simulation.flight_controller.read_state("gomspace.output.output2"),
+            simulation.flight_controller.read_state("gomspace.output.output3"),
+            simulation.flight_controller.read_state("gomspace.output.output4"),
+            simulation.flight_controller.read_state("gomspace.output.output5"),
+            simulation.flight_controller.read_state("gomspace.output.output6")]
+
+        for n in range(0, len(output)):
+            out_n = output[n]
+            if out_n < 0.5 or out_n > 3.0:
+                print("Output-" + n +
+                      " is out of range [0.3, 0.5] at: " + out_n)
+
+        wdt_i2c_time_left = simulation.flight_controller.read_state(
+            "gomspace.wdt_i2c_time_left")
+        counter_wdt_i2c = simulation.flight_controller.read_state(
+            "gomspace.counter_wdt_i2c")
+        counter_boot = simulation.flight_controller.read_state(
+            "gomspace.counter_boot")
```

Comment:
We want to check that the time left is greater than 99 (the WDT timer is currently configured for 100 seconds)

We can't really write tests for the other two values, maybe just print them out to the screen?